lib/node_modules/@stdlib/blas/base/dsdot/README.md

```diff
@@ -227,6 +227,32 @@ The function accepts the following arguments:
 double c_dsdot( const CBLAS_INT N, const float *X, const CBLAS_INT strideX, const float *Y, const CBLAS_INT strideY );
 ```
 
+#### c_dsdot_ndarray( N, \*X, strideX, offsetX, \*Y, strideY, offsetY )
+
+Computes the dot product of two single-precision floating-point vectors with extended accumulation and result and using alternative indexing semantics.
+
+```c
+const float x[] = { 4.0f, 2.0f, -3.0f, 5.0f, -1.0f };
+const float y[] = { 2.0f, 6.0f, -1.0f, -4.0f, 8.0f };
+
+double v = c_dsdot_ndarray( 5, x, 1, 0, y, 1, 0 );
+// returns -5.0
+```
+
+The function accepts the following arguments:
+
+-   **N**: `[in] CBLAS_INT` number of indexed elements.
+-   **X**: `[in] float*` first input array.
+-   **strideX**: `[in] CBLAS_INT` index increment for `X`.
+-   **offsetX**: `[in] CBLAS_INT` starting index for `X`.
+-   **Y**: `[in] float*` second input array.
+-   **strideY**: `[in] CBLAS_INT` index increment for `Y`.
+-   **offsetY**: `[in] CBLAS_INT` starting index for `Y`.
+
+```c
+double c_dsdot_ndarray( const CBLAS_INT N, const float *X, const CBLAS_INT strideX, const CBLAS_INT offsetX, const float *Y, const CBLAS_INT strideY, const CBLAS_INT offsetY );
+```
+
```

Comment:
@aman-095 You updated `example.c`, but not the example below. Was that intentional?

Reply:
No, @kgryte it was not intentional, I think I missed that. Updated.